budcamp/Homework2/Homework2/Point.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework2
+{
+    struct Point
+    {
+        public double X { get; set; }
+        public double Y { get; set; }
+
+        public Point()
+        {
+            X = 0;
+            Y = 0;
+        }
+
+        public Point(int x, int y)
+        {
+            this.X = x;
+            this.Y = y;
+        }
+
+        public bool TryParsePoint(string input, out Point point)
+        {
+            point = default(Point);
+            int num1;
+            int num2;
+            string[] splitedString = new string[2];
+
+            try
+            {
+                splitedString = input.Split(',');
```

Comment:
you can remove the declaration of `splitedString` from above and do it here.

Reply:
done

Reply:
but how i have to think? why i have to declarae it int th try-catch block?

Reply:
why not? if you can declare the value where you plan to initialize it - do it.

Reply:
ty